### DISCUSSION TITLE
Versioning of Documentation

### DISCUSSION THREAD

**Discussion by simonmoedinger** [vuepress-theme-hope/vuepress-theme-hope#2947] [Q&A] [answered]
Hi I am currently building a documentation and use the theme from mr hope. I need a versioning in the docu to separate the content of the different releases. Like for example vuepress1.0.0 and vuepress 2.0.0 in their navbar [navbar ](https://v2.vuepress.vuejs.org/reference/node-api.html#options).

Is there a plugin that solves this? Or does anyone have a good blog/site on how to implement this. 

Thanks a lot

**Comment by Mister-Hope** [comment, marked as answer]
My suggestions is splitting them into different docs.
